apps/coscrad-frontend/src/components/footer/footer.tsx

```diff
@@ -22,6 +48,35 @@ export const Footer = (): JSX.Element => {
                 backgroundColor: '#ededed',
             }}
         >
+            <Grid item>
+                Phone: {phoneNumber}
+                Address: {address}
+            </Grid>
+            <Grid item sx={{ textAlign: 'center' }}>
+                {socialMediaIcons.map(({ link, icon }) =>
+                    link ? (
+                        <IconButton
+                            href={link}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            color="primary"
+                            children={icon}
+                        />
+                    ) : (
+                        ''
+                    )
+                )}
+            </Grid>
+            <Grid item sx={{ textAlign: 'center' }}>
+                {internalLinks.map(({ description, url, iconUrl }) => (
+                    <Tooltip title={description}>
+                        <a href={url} target="_blank" rel="noopener noreferrer">
+                            <img height={50} src={iconUrl} alt={description} />
```

Comment:
Be sure to experiment with a wide variety of image dimensions on your config when manual testing.